gradlew

```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m"'
```

Comment:
@eandr127 is this setting the java heap memory to 64 megabytes? Wouldn't we not want to do this?

Reply:
I checked [here](https://gradle.org/whats-new/gradle-5/#more-memory-efficient-gradle-execution), and it's just the new default setting for the command line client, so it shouldn't make any difference on the robot.

Reply:
Alright, as long as it wasn't accidental :+1:.